identify.go

```diff
@@ -14,8 +14,9 @@ type Identify struct {
 	MessageId    string       `json:"messageId,omitempty"`
 	AnonymousId  string       `json:"anonymousId,omitempty"`
 	UserId       string       `json:"userId,omitempty"`
-	Timestamp    time.Time    `json:"timestamp,omitempty"`
+	Timestamp    time.Time    `json:"originalTimestamp,omitempty"`
 	Context      *Context     `json:"context,omitempty"`
+	Properties   Properties   `json:"properties,omitempty"`
```

Comment:
`identify` will not contain `Properties`

Reply:
Thanks for looking at this!

Are the docs at https://www.rudderstack.com/docs/event-spec/standard-events/common-fields/ incorrect? They state that `properties` is one of the common fields on all events?

Reply:
`properties` is optional. It is not relevant for `identify`. `Identify` accepts traits only.

Reply:
Ok, I've removed `properties` from `Identify`. I still find it a bit confusing that the docs say `properties` is a valid field for all events. I guess it's "allowed" but just not actually used in the case of `Identify`?